packages/config/skills/turborepo/SKILL.md

```
@@ -0,0 +1,311 @@
+
+```
+Enforce boundaries?
+├─ Check for violations → turbo boundaries
+├─ Tag packages → references/boundaries/README.md#tags
+├─ Restrict which packages can import others → references/boundaries/README.md#rule-types
+└─ Prevent cross-package file imports → references/boundaries/README.md
+```
+
+### "I need to manage dependencies"
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)